maven_iuvs/integration.py

```diff
@@ -255,8 +255,17 @@ def fit_line(myfits, wavelength,
 
         for ispa in range(n_spa):
             waves = myfits['Observation'].data['WAVELENGTH'][0, ispa]
-            DN = myfits['detector_dark_subtracted'].data[iint, ispa]
-            DN_unc = myfits['Random_dn_unc'].data[iint, ispa]
+
+            try:
+                DN = myfits['detector_dark_subtracted'].data[iint, ispa] # l1b
+            except:
+                DN = myfits['Primary'].data[iint, ispa] # l1a
+
+            try: 
+                DN_unc = myfits['Random_dn_unc'].data[iint, ispa] # l1b
+            except:
+                DN_unc = np.zeros_like(DN) # l1a -- this is wrong obviously
```

Comment:
better to fill with NaN

Reply:
Fixed by 1669a94a7aa155787effe1d66b032aecf75395d3